packages/cosmic-swingset/Makefile

```diff
@@ -121,8 +144,27 @@ t1-provision-one: wait-for-cosmos
 		  --gas=auto --gas-adjustment=1.2 --broadcast-mode=block --yes --chain-id=$(CHAIN_ID) \
 		  t1/$(BASE_PORT) $$addr | tee /dev/stderr | grep -q '"code":0'; }
 
+t1/$(BASE_PORT)/cosmos-client-account.setup:
+	test ! -f t1/$(BASE_PORT)/cosmos-client-account || \
```

Comment:
It's not something I know how to deal with, but this Makefile has grown a lot (sprouting all sorts of `sh`-isms like extra mutant eyeballs.. kinda threatening to crawl out of its cage at this point).. we should think about a better way to manage the things it's being asked to manage. Maybe some new subcommands of the `agoric` tool?

Reply:
Most of the things this Makefile is doing would want to be moved to integration tests.